Kitodo/src/test/java/org/kitodo/production/services/file/FileServiceIT.java

```diff
@@ -48,12 +61,61 @@ public void testRenameFileWithLockedFile() throws IOException {
             fileService.renameFile(oldUri, "newName.xml");
         } catch (IOException e) {
             URI newUri = URI.create("fileServiceTest/newName.xml");
-            Assert.assertFalse(fileService.fileExist(newUri));
-            Assert.assertTrue(fileService.fileExist(oldUri));
+            assertFalse(fileService.fileExist(newUri));
+            assertTrue(fileService.fileExist(oldUri));
 
         } finally {
             outputStream.close();
         }
 
     }
+
+    @Test
+    public void testCreateDirectories() throws IOException {
+        Path testBaseDirectoryPath = Paths.get(ConfigCore.getParameter(ParameterCore.DIR_XML_CONFIG));
+
+        // delete existing directories so that directories are actually created
+        Path firstDirectoryOfTestPath = testBaseDirectoryPath.resolve(SEVERAL);
+        if (Files.isDirectory(firstDirectoryOfTestPath)) {
+            TreeDeleter.deltree(firstDirectoryOfTestPath);
+        }
+
+        String testBaseDirectoryURI = testBaseDirectoryPath.toUri().toString();
+        String createDirectories = "several/directories/can/be/created/at/once";
+        new FileService().createDirectories(URI.create(testBaseDirectoryURI + createDirectories));
+        assertTrue(Files.isDirectory(firstDirectoryOfTestPath.resolve("directories/can/be/created/at/once")));
+
+        // clean up
+        TreeDeleter.deltree(firstDirectoryOfTestPath);
+    }
+
+    @Test
+    public void testCreateDirectoriesWithTrailingSlash() throws IOException {
+        Path testBaseDirectoryPath = Paths.get(ConfigCore.getParameter(ParameterCore.DIR_XML_CONFIG));
+
+        // delete existing directories so that directories are actually created
+        Path firstDirectoryOfTestPath = testBaseDirectoryPath.resolve(SEVERAL);
```

Comment:
I think the testenvironment should ensure, that there is no such directory. This part should be removed.